dune-project

```
@@ -0,0 +1 @@
+(lang dune 2.0)
```

Used by (in order) dune-release and dune’s META file generation
```suggestion
(lang dune 2.0)
(name lz4)
(version 1.1.1)
```